src/Symfony/Bundle/WebProfilerBundle/Controller/ProfilerController.php

```diff
@@ -11,15 +11,20 @@
 
 namespace Symfony\Bundle\WebProfilerBundle\Controller;
 
+use Symfony\Bundle\FrameworkBundle\Client;
+use Symfony\Bundle\FrameworkBundle\DataCollector\RequestDataCollector;
```

Comment:
your phpdoc is wrong. The `request` panel may be the one from the component. It is not always the extended version of FrameworkBundle